packages/sanity/src/desk/panes/documentList/DocumentListPaneHeader.tsx

```diff
@@ -57,7 +57,15 @@ export const DocumentListPaneHeader = memo(
         }
         backButton={
           features.backButton &&
-          index > 0 && <Button as={BackLink} data-as="a" icon={ArrowLeftIcon} mode="bleed" />
+          index > 0 && (
+            <Button
+              as={BackLink}
+              data-as="a"
+              icon={ArrowLeftIcon}
+              mode="bleed"
+              tooltipProps={{content: 'Back'}}
```

Comment:
This currently doesn't render correctly, please verify once the `alpha` branch of Sanity UI is up to date

Reply:
Looks correct (positioned at the bottom) after last fallback placements pr update